src/isIsogram.test.js

```diff
@@ -1,9 +1,41 @@
-'use strict';
+"use strict";
 
-describe('isIsogram', () => {
-  const { isIsogram } = require('./isIsogram');
+describe("isIsogram", () => {
+  const { isIsogram } = require("./isIsogram");
 
-  it(`should be declared`, () => {
+  it("should be declared", () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it("should throw error if argument not provided", () => {
+    try {
+      isIsogram();
+    } catch (error) {
+      expect(error).toBeInstanceOf(Error);
+    }
+  });
+
+  it("should return boolean value", () => {
+    expect(isIsogram("")).toBeInstanceOf(Boolean);
+  });
+
+  it("should return true if value equal empty string", () => {
+    expect(isIsogram("")).toBe(true);
+  });
+
+  it("should return true to doubled chars", () => {
+    expect(isIsogram("aA")).toBe(false);
+  });
+
+  it("should be not case sensitive ", () => {
+    expect(isIsogram("a")).toEqual(isIsogram("A"));
```

Comment:
1) u don't need to call the function twice for this, u can use the following syntax
```suggestion
    expect(isIsogram("aka")).toBe(false));
```
2) u can use another syntax for `Truthy` and `Falsy`
```suggestion
    expect(isIsogram("aka")).toBeFalsy());
```